pages/HC/operations.vue

```diff
@@ -101,44 +126,63 @@ import axios from 'axios'
           los:'',
           sublos:'',
           bottomlos:'',
-          status:'',
+          category:'',
+          status:'false',
           titles:'',
-          region: '',
+          location:'',
           date1: '',
-          date2: '',
-          delivery: false,
-          type: [],
-          resource: '',
-          desc: ''
+          date2: 'false'
+        },
+         rules: {
+          name: [
+            { required: true, message: 'Please input Employee name and surname', trigger: 'blur' },
+           // { min: 3, max: 5, message: 'Length should be 3 to 5', trigger: 'blur' }
+          ],
+          date1: [
```

Comment:
Instead of using `date1`, `date2`, you should have a better naming convention.

Reply:
I have been created new variable as Hire_Date and Expire_Date ,Instead of using date1, date2. Thank you for your suggestion.